### PR TITLE
fix: Drive returns the wrong tier

### PR DESCRIPTION
**Pull request by adrien-coye**
- Fix `isFreePack` property
- Integration test for `Drive` object.
- Shared JSON loader in test target based on https://github.com/Infomaniak/ios-kDrive/pull/1084 is merged.

### CODE REVIEW

File: kDriveTests/kDriveCore/Drive/ITDrive.swift [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ */
+
+import kDriveCore
+import XCTest
+
+/// Integration tests for Drive
+final class ITDrive: XCTestCase {
+    // MARK: - Parsing
+
+    // TODO: Move to a shared helper
```
Comment (adrien-coye) [inline kDriveTests/kDriveCore/Drive/ITDrive.swift:26]: TODO: factorise this JSON loader code.

File: kDriveTests/kDriveCore/Drive/ITDrive.swift [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    private func freeDriveJson() -> Data? {
+        data(forResource: "free_drive", withExtension: "json")
+    }
+
+    private func paidDriveJson() -> Data? {
+        data(forResource: "paid_drive", withExtension: "json")
+    }
+
+    func testFreeDriveIsFree() {
```
Comment (adrien-coye) [inline kDriveTests/kDriveCore/Drive/ITDrive.swift:44]: With some more time I can now add way more assertions for extra security.

**Review by PhilippeWeidmann — DISMISSED** [review]

File: kDriveTests/kDriveCore/Drive/ITDrive.swift (hunk 1 above)
Comment (adrien-coye) [reply kDriveTests/kDriveCore/Drive/ITDrive.swift]: done

File: kDriveTests/kDriveCore/Drive/ITDrive.swift (hunk 2 above)
Comment (adrien-coye) [reply kDriveTests/kDriveCore/Drive/ITDrive.swift:34]: Will do later

**Comment by adrien-coye** [comment]
Discussed with @PhilippeWeidmann , merging now.

**Pull request merged**